app/main.py

```diff
@@ -2,0 +1,19 @@
+from app.errors import NotWearingMaskError, VaccineError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+
+    masks_to_buy = 0
+    for person in friends:
+        try:
+            cafe.visit_cafe(person)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
Code Style: The error message here is not very descriptive. Consider changing it to something more informative, such as 'Friend {name} is not vaccinated.'